packages/rich-text/src/test/get-format-types.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getFormatTypes } from '../get-format-types';
+import { unregisterFormatType } from '../unregister-format-type';
+import { registerFormatType } from '../register-format-type';
+
+describe( 'getFormatTypes', () => {
+	// Initialize format store.
+	require( '../store' );
+
+	afterEach( () => {
+		getFormatTypes().forEach( ( format ) => {
+			unregisterFormatType( format.name );
+		} );
+	} );
+
+	it( 'should return an empty array at first', () => {
+		expect( getFormatTypes() ).toEqual( [] );
+	} );
+
+	it( 'should return all registered formats', () => {
+		const formatType1 = { edit: noop, title: 'format title' };
```

Comment:
I personally like more descriptive names for variables.
How about:
- `testFormat`
- `testFormatWithSettings`
to align with the names used as first param in `registerFormatType`?